app/controllers/works_controller.rb

```diff
@@ -192,7 +192,8 @@ def new_work_form
       agree_to_terms: current_user.agree_to_terms?,
       contact_emails_attributes: [{ email: current_user.email_address }],
       work_type: @collection.work_type,
-      work_subtypes: @collection.work_subtypes
+      work_subtypes: @collection.work_subtypes,
+      deposits_contact_email: @collection.deposits_contact_email
```

Comment:
Make sure the new work form has the collection's contact email for deposits on it.

Reply:
Will do.